src/winscard_clnt.c

```diff
@@ -498,6 +491,40 @@ LONG SCardEstablishContext(DWORD dwScope, LPCVOID pvReserved1,
 	return rv;
 }
 
+static list_t contextMapList;
+
+CONSTRUCTOR static void constructor() {
+	int lrv;
+
+	lrv = list_init(&contextMapList);
+	if (lrv < 0)
+	{
+		Log2(PCSC_LOG_CRITICAL, "list_init failed with return value: %d",
+			lrv);
+		return SCARD_E_NO_MEMORY;
+	}
+
+	lrv = list_attributes_seeker(&contextMapList,
+			SCONTEXTMAP_seeker);
+	if (lrv <0)
+	{
+		Log2(PCSC_LOG_CRITICAL,
+			"list_attributes_seeker failed with return value: %d", lrv);
+		list_destroy(&contextMapList);
+		return SCARD_E_NO_MEMORY;
+	}
+
+	if (getenv("PCSCLITE_NO_BLOCKING"))
+	{
+		Log1(PCSC_LOG_INFO, "Disable shared blocking");
+		sharing_shall_block = FALSE;
+	}
+}
+
+DESTRUCTOR static void destructor() {
```

Comment:
I propose to just add this destructor() function.

Reply:
f